rideshare/LoginViewController.swift

```diff
@@ -20,12 +20,19 @@ class LoginViewController: UIViewController {
     }
     
     @IBAction func onSignIn(_ sender: Any) {
+        
         PFUser.logInWithUsername(inBackground: emailField.text!, password: passwordField.text!) {
             (user: PFUser?, error: Error?) -> Void in
+            
             if user != nil {
                 self.transitionToMain()
             } else {
                 print("Error: \(String(describing: error?.localizedDescription))")
+                let alert = UIAlertController(title: "Error", message: error?.localizedDescription, preferredStyle: .alert)
+                
+                alert.addAction(UIAlertAction(title: "Try Again", style: .default, handler: nil))
```

Comment:
Same

Reply:
Also, here you can say a meaningful message like "There was an error signing in, please try again." or something similar